testing/hamlib/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Bradford D. Boyle <bradford.d.boyle@gmail.com>
+# Maintainer: Bradford D. Boyle <bradford.d.boyle@gmail.com>
+pkgname=hamlib
+pkgver=3.0
+pkgrel=0
+pkgdesc="Library to control radio transceivers and receivers"
+url="http://www.hamlib.org/"
+arch="all"
+license="LGPLv2+"
+depends="perl python"
+depends_dev=""
+makedepends="$depends_dev linux-headers perl-dev python-dev swig tcl-dev"
```

Comment:
I see there are perl, python and tcl bindings.  These should be separated into subpackages so that installing hamlib does not pull in perl, python and TCL from the start.